tasks/pip.yml

```diff
@@ -58,7 +58,7 @@
 
 - name: Install uwsgi
   pip:
-    name: "{{ item }}"
+    name: http://http.debian.net/debian/pool/main/u/uwsgi/uwsgi_2.0.14+20170111.orig.tar.gz
```

Comment:
Should i create a new section just for debian or is there a shorter/better way to make such a change ?